### ISSUE TITLE
name[casing]: False positive when Chinese task/play names are used

### ISSUE THREAD

**Issue by brlin-tw** [ansible/ansible-lint#2340]
##### Summary

The following error is triggered in our integration tests:

```
playbooks/ping.yml:6 Task/Handler: 檢查被控端是否可以連接
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
$ ansible --version
ansible [core 2.13.2]
  config file = /media/veracrypt1/文件/中信安科技/解決方案/自動化 Automation/Ansible 自動化配置工具/Ansible 角色範本/ansible.cfg
  configured module search path = ['/home/brlin/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/brlin/.local/pipx/venvs/ansible/lib/python3.10/site-packages/ansible
  ansible collection location = __REDACTED__/playbooks/collections
  executable location = /home/brlin/.local/bin/ansible
  python version = 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0]
  jinja version = 3.1.2
  libyaml = True

$ ansible-lint --version
WARNING: PATH altered to include /home/brlin/.local/pipx/venvs/ansible/bin
ansible-lint 6.5.1 using ansible 2.13.2
```

- ansible installation method: pip(pipx)
- ansible-lint installation method: pip(pipx)

##### OS / ENVIRONMENT

OS: Ubuntu 22.04

##### STEPS TO REPRODUCE

```yaml
# playbooks/ping.yml
- name: 測試範例(PoC)
  hosts: all
  tasks:
    - name: 檢查被控端是否可以連接
      ping:
```

##### Desired Behavior

When `name` is start with character that is not in a locale that has the casing concept, skip the rule.

##### Actual Behavior

```paste below
$ ansible-lint playbooks/ping.yml 
WARNING: PATH altered to include /home/brlin/.local/pipx/venvs/ansible/bin
WARNING  Listing 2 violation(s) that are fatal
name: All names should start with an uppercase letter. (name[casing])
playbooks/ping.yml:2

name: All names should start with an uppercase letter. (name[casing])
playbooks/ping.yml:5 Task/Handler: 檢查被控端是否可以連接


Finished with 0 failure(s), 2 warning(s) on 1 files.

```

**Comment by ssbarnea** [comment]
For the moment you can add `name[casing]` to ignore list, which will disable this check. We need to decide if we want to just document this or alter the implementation to make it really trigger when first character is lowercase (as opposed to checking for uppercase presence). Probably we will include a fix in the next patch. Thanks for reporting it.

**Comment by karras** [comment]
@ssbarnea I recently also stumbled into the capital letter warnings after upgrading to 6.5.x. 

When checking the related PR from the COP (https://github.com/redhat-cop/automation-good-practices/pull/64/commits) I noticed that it seems the capital letter recommendation was in the end removed again:

https://github.com/redhat-cop/automation-good-practices/pull/64/commits/bb9df441bb375427101af68e61403b462aecfec0

Thus would it make sense to disable the check by default to be aligned with the COP?

Thanks for all your work around ansible-lint!
